aiidalab_widgets_base/structures.py

```diff
@@ -842,15 +869,88 @@ def inner(ref, *args, **kwargs):
                 ref,
                 *args,
                 **kwargs,
-                atoms=ref.structure.copy(),
                 selection=ref.selection.copy(),
             )
 
     return inner
 
 
+class BasicCellEditor(ipw.VBox):
+    """Widget that allows for the basic cell editing."""
+
+    structure = Instance(Atoms, allow_none=True)
+
+    def __init__(self, title=""):
+        self.title = title
+
+        # cell transfor opration widget
+        primitive_cell = ipw.Button(
+            description="Convert to primitive cell",
+            layout={"width": "initial"},
+        )
+        primitive_cell.on_click(self.def_primitive_cell)
+
+        conventional_cell = ipw.Button(
+            description="Convert to conventional cell",
+            layout={"width": "initial"},
+        )
+        conventional_cell.on_click(self.def_conventional_cell)
+
+        super().__init__(
+            children=[
+                ipw.HTML(
+                    "<b>Cell transformation:</b>",
+                    layout={"margin": "20px 0px 10px 0px"},
+                ),
+                ipw.HBox(
+                    [
+                        primitive_cell,
+                        conventional_cell,
+                    ],
+                ),
+            ],
+        )
+
+    @_register_structure
+    def def_primitive_cell(self, _=None, atoms=None):
+        atoms = self._to_standardize_cell(atoms, to_primitive=True)
+
+        self.structure = atoms
+
+    @_register_structure
+    def def_conventional_cell(self, _=None, atoms=None):
+        atoms = self._to_standardize_cell(atoms, to_primitive=False)
+
+        self.structure = atoms
+
+    @staticmethod
+    def _to_standardize_cell(
```

Comment:
I think it should be called `to_standard_cell` or `stardardize_cell`.

Reply:
Fair point, I use `to_standard_cell`.

Reply:
I think the function is now called `to_standardize_cell`. I am confused :)

Reply:
Ah, my bad. Can not focus on the so early train. I change it to `to_standard_cell`.